docs/diagrams/SetThemeSequenceDiagram.puml

```diff
@@ -0,0 +1,36 @@
+@startuml
+!include style.puml
+
+box Ui UI_COLOR_T1
+participant ":MainWindow" as MainWindow UI_COLOR
+end box
+
+box Ui
+participant "primaryStage:Stage" as Stage #darkgrey
+
+
+box Ui LOGIC_COLOR_T1
+participant ":Scene" as Scene LOGIC_COLOR
+end box
+
+
+
+[-> MainWindow: handleDark()
+activate MainWindow
+MainWindow -> Stage : getScene()
+activate Stage
+Stage -> Scene : getStyleSheets()
```

Comment:
Feels slightly inaccurate to what actually happens. When you use `getScene()`, it returns the scene immediately, and you use that scene to call `getStyleSheets()`, rather than the `getScene()` method calling `getStyleSheets()`.
![image](https://user-images.githubusercontent.com/97277891/199952826-4b4666cc-ec6e-4773-a9d3-fecb46af6458.png)
Pardon the bad paint drawing, but I think something like this would be more accurate. The drawing is not complete, you'll have to complete the whole thing.

Reply:
yes you're right thanks